contracts/ssnlist.scilla

```diff
@@ -914,11 +919,10 @@ transition ChangeBNumReq(input_bnum_req: Uint128, initiator: ByStr20)
   bnum_req := input_bnum_req
 end
 
-transition UpdateContractAddr(proxy_addr: ByStr20, gzil_addr: ByStr20, initiator: ByStr20)
+transition UpdateGzilAddr(gzil_addr: ByStr20, initiator: ByStr20)
```

Comment:
Why keep proxy_address as an immutable parameter then?

Reply:
Updated to inmutable parameter.